packages/utils/src/validator/isInRange/index.ts

```diff
@@ -0,0 +1,32 @@
+interface IsInRangeProps {
+  value: number;
+  min: number;
+  max: number;
+  equalOptions?: {
+    min?: boolean;
+    max?: boolean;
+  };
+}
+
+export const isInRange = ({
+  value,
+  min,
+  max,
+  equalOptions,
+}: IsInRangeProps) => {
+  if (min == null) throw new Error('min값은 필수입니다.');
+  if (max == null) throw new Error('max값은 필수입니다.');
+  if (min > max) throw new Error('min은 max보다 작아야합니다.');
+
+  const defaultEqualOptions = {
+    min: true,
+    max: false,
+  };
+
+  const mergedEqualOptions = { ...defaultEqualOptions, ...equalOptions };
+
+  const minCheck = mergedEqualOptions.min ? value >= min : value > min;
+  const maxCheck = mergedEqualOptions.max ? value <= max : value < max;
+
+  return minCheck && maxCheck;
+};
```

Comment:
```suggestion
export const isInRange = ({
  value,
  min,
  max,
  equalOptions = {},
}: IsInRangeProps) => {
  if (min == null || max == null) {
    throw new Error('The min and max values are invalid.');
  }
  if (min > max) {
    throw new Error('The min value cannot be greater than the max value.');
  }

  const { min: minEqual = true, max: maxEqual = false } = equalOptions;

  const isWithinMin = minEqual ? value >= min : value > min;
  const isWithinMax = maxEqual ? value <= max : value < max;

  return isWithinMin && isWithinMax;
};
```

- `min`, `max`의 `== null`검증은 통합해서 좋을 것 같습니다..!
- `mergedEqualOptions`를 하신 이유가 있을까요!? 전개연산자로 리소스를 들이기 보다 아래와 같이 기본 값을 설정해도 동일하게 동작합니다 🤗
```ts
  const { min: minEqual = true, max: maxEqual = false } = equalOptions;
```
- boolean의 의미로 `isWithinMin`, `isWithinMin`의 네이밍은 어떨까요!?

Reply:
@ssi02014 좋은 의견 감사합니다!

mergedEqualOptions의 경우 디폴트를 유지하면서 받아온 값을 유지해야한다는 생각을 먼저 했다보니 그 방식이 코드에 나온것같네요! 😂 제안해주신 방식이 훨씬 간결하여 해당 방향으로 수정해두겠습니다! 

수정하면서 함께 제안해주신 Error throw 케이스 통합 및 message 영문변경도 같이 적용해두겠습니다! 감사합니다! 👍